Sources/Host/FreeBSD/ProcStat.cpp

```diff
@@ -8,15 +8,16 @@
 // PATENTS file in the same directory.
 //
 
-#include <libprocstat.h>
-#include <libutil.h>
-#include <string>
 #include <sys/elf.h>
 #include <sys/param.h>
 #include <sys/sysctl.h>
 #include <sys/types.h>
 #include <sys/user.h>
 
+#include <libprocstat.h>
+#include <libutil.h>
+#include <string>
```

Comment:
I understand the problem you're trying to solve but I'm not sure this is the best way to do it. According to https://www.freebsd.org/cgi/man.cgi?query=libprocstat&sektion=3&apropos=0&manpath=FreeBSD%2010.0-RELEASE `sys/param.h` and `sys/queue.h` are required for `libprocstat.h` and only `sys/param.h` is required for `libutil.h`.

What you should do instead is to sort all the headers properly the way they were before, and then pull up `sys/param.h` and `sys/queue.h` before `libprocstat.h` and `libutil.h` and enclose these headers with `// clang-format off` and `// clang-format on` to make sure they dont' get automatically reordered in the future.

So you'd end up with something like this:
```
// clang-format off
#include <sys/param.h>
#include <sys/queue.h>
#include <libprocstat.h>
#include <libutil.h>
// clang-format on
#include <string>
#include <sys/elf.h>
#include <sys/param.h>
#include <sys/sysctl.h>
#include <sys/types.h>
#include <sys/user.h>
```